vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -158,6 +158,7 @@ func handleRootHealthz(checks ...HealthzChecker) http.HandlerFunc {
 		}
 		// always be verbose on failure
 		if failed {
+			glog.Warningf("%vhealthz check failed", verboseOut.String())
```

Comment:
the formatting is weird here ;-)

Reply:
same as the /healthz body, which looks good.